src/components/dashboard/components/TaskList.js

```diff
@@ -21,7 +21,7 @@ const TaskList = props => {
       </ListHeader>
       {props.events.map(event => {
         return (
-          <ListItem onClick={() => handleClick(event.id)} key={event.id}>
+          <ListItem className="task-item" onClick={() => handleClick(event.id)} key={event.id}>
```

Comment:
this line is about to change as new feature will be merged (background color of task for completed and past due)